### PR TITLE
Add recipe for installing Anaconda 2.3.0.

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

**Comment by gregvonkuster** [comment]
The initial recipe I committed was a version I did not mean to commit (sorry).  The version I meant to commit is in my 2nd change in this PR.  Only the PATH environment variable needs to be modified for an Anaconda environment so that it gets the right Python from the ~/INSTALL_DIR/bin directory.

**Comment by gregvonkuster** [comment]
ping @martenson @bgruening Can this be merged?

File: packages/package_anaconda_2_3_0/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </install>
+        <readme>
+            <![CDATA[
+This installation recipe installs the Anaconda version 2.3.0 of the Python programming language
+version 2.7.10.  Anaconda is a completely free Python distribution (including for commercial use
+and redistribution). It includes more than 300 of the most popular Python packages for science,
+math, engineering, and data analysis.  This list of packages is available at
+http://docs.continuum.io/anaconda/pkg-docs.
+
+Care should be taken in using the Anaconda Python environment that is installed by this recipe.
```
Comment (bgruening) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: Can we add this sentence also to the shed.yml

Comment (gregvonkuster) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: It's now in there!  Thanks!

File: packages/package_anaconda_2_3_0/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="anaconda" version="2.3.0">
+        <install version="1.0">
+            <actions_group>
+                <actions os="linux" architecture="x86_64">
+                    <action type="download_file" target_filename="Anaconda-2.3.0-Linux-x86_64.sh">https://3230d63b5fc54e62148e-c95ac804525aac4b6dba79b00b39d1d3.ssl.cf1.rackcdn.com/Anaconda-2.3.0-Linux-x86_64.sh</action>
```
Comment (martenson) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: please add md5sum as in https://github.com/galaxyproject/tools-iuc/blob/84bf03549abc3f1f7fa0bf5a9fec91ae2c8a0d36/packages/package_bz2file_0_98/tool_dependencies.xml#L6

Comment (gregvonkuster) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: @martenson What is the process you are using for generating these?

Comment (martenson) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: I use
```
$ md5 hoho.txt
MD5 (hoho.txt) = d52825c4b3939dc565a403e8895093fa
$ which md5
/sbin/md5
```

Comment (martenson) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: **update** I think it should be checking it here too, see:
https://github.com/galaxyproject/galaxy/blob/dev/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py#L203

File: packages/package_anaconda_2_3_0/tool_dependencies.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                    <action type="shell_command">bash Anaconda-2.3.0-Linux-x86_64.sh -b -f -p $INSTALL_DIR</action>
+                </actions>
+                <actions os="darwin" architecture="x86_64">
+                    <action type="download_file" target_filename="Anaconda-2.3.0-MacOSX-x86_64.sh">https://3230d63b5fc54e62148e-c95ac804525aac4b6dba79b00b39d1d3.ssl.cf1.rackcdn.com/Anaconda-2.3.0-MacOSX-x86_64.sh</action>
```
Comment (bgruening) [inline packages/package_anaconda_2_3_0/tool_dependencies.xml]: @gregvonkuster just for my understanding, why are you using donwload_file and not download_by_url?

**Comment by martenson** [comment]
Bringing up @bgruening's question as it got hidden by github:
@gregvonkuster just for my understanding, why are you using donwload_file and not download_by_url?

**Comment by gregvonkuster** [comment]
I'm using download_file because this recipe downloads a command line shell script that performs the installation.  It's been quite a while since I originally wrote this, but if I remember correctly, using the download_by_url action on the shell script did not behave correctly.

**Comment by gregvonkuster** [comment]
@martenson  The md5 checks are now in there.  Thanks!

**Comment by gregvonkuster** [comment]
@bgruening @martenson I just tested again and download_by_url works fine, so I've updated the recipe to use that.  Thinking back, I probably based this recipe on another I was writing that had these installations as steps below an initial download_by_url, but I can't be sure.  Anyway, the recipe now uses it.

**Comment by martenson** [comment]
thanks a lot @gregvonkuster !

**Comment by gregvonkuster** [comment]
@martenson @bgruening Thanks for the help and merge!  ;)

**Pull request merged**